README.md

```diff
@@ -106,13 +111,13 @@ To have default account(s) seeded at runtime:
 
 In Development:
 ```bash
-MONGO_SEED=true grunt
+MONGO_SEED=true npm start
 ```
 It will try to seed the users 'user' and 'admin'. If one of the user already exists, it will display an error message on the console. Just grab the passwords from the console.
 
 In Production:
 ```bash
-MONGO_SEED=true grunt prod
+MONGO_SEED=true npm start:prod
```

Comment:
This may be out of the scope of this PR, but I feel that the `MONGO_SEED` option here is a bit confusing. I think we could just move the seeding option to it's own section.